pkg/gatewayserver/scheduling/scheduler.go

```diff
@@ -252,32 +252,45 @@ var (
 	errNoServerTime          = errors.DefineAborted("no_server_time", "no server time")
 )
 
+// Options define options for scheduling downlink.
+type Options struct {
+	PayloadSize int
+	ttnpb.TxSettings
+	RTTs        RTTs
+	Priority    ttnpb.TxSchedulePriority
+	UplinkToken *ttnpb.UplinkToken
+}
+
 // ScheduleAt attempts to schedule the given Tx settings with the given priority.
 // If there are round-trip times available, the maximum value will be used instead of ScheduleTimeShort.
-func (s *Scheduler) ScheduleAt(ctx context.Context, payloadSize int, settings ttnpb.TxSettings, rtts RTTs, priority ttnpb.TxSchedulePriority) (Emission, error) {
+func (s *Scheduler) ScheduleAt(ctx context.Context, opts Options) (Emission, error) {
 	defer trace.StartRegion(ctx, "schedule transmission").End()
 
 	s.mu.RLock()
 	defer s.mu.RUnlock()
 	if !s.clock.IsSynced() {
-		return Emission{}, errNoClockSync.New()
+		if token := opts.UplinkToken; token != nil {
```

Comment:
Maybe good to also check `!token.ServerTime.IsZero()`, just to catch the edge case where uplink and downlink GS are not the same version.